Formula/jhiccup.rb

```diff
@@ -1,7 +1,7 @@
 class Jhiccup < Formula
   desc "Measure pauses and stalls of an app's Java runtime platform"
-  homepage "http://www.jhiccup.com"
-  url "http://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
+  homepage "http://www.jhiccup.com/"
+  url "https://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
```

Comment:
looks like there is a new version, so this one probably belongs in a separate PR